docs/project-configuration/builds.mdx

```diff
@@ -24,16 +24,19 @@ seoFrontMatterUpdated: true
 ## Introduction
 
 A supergraph build is a fully-functional, immutable [supergraph](/basics/core-concepts.mdx#supergraph) API which is
-built based on your project's configuration.
+generated based on your project's configuration.
 
-Supergraph builds can be used to incrementally test your project's API.
+Supergraph builds can be used to incrementally test your project's API with a unique build URL. 
 
 While there can be many supergraph builds for a project, only one can be **applied** at a time. The applied supergraph
 build is the one that is currently serving your API from the project's endpoint.
 
-A supergraph build is not created in your local directory, but is rather created and stored in the Hasura DDN service.
+You can create a supergraph build locally or in the cloud on Hasura DDN.
+
 They are lightweight and can be created as often as you like to check your changes.
 
+The format of a build is not something you will need to be concerned with, or need to edit, but it is set of large JSON objects which describes the supergraph.
```

Comment:
On first read, this made me feel like:

> You're not smart enough to understand what a build is...don't worry about it.

To we need to say this? Could we get away with saying something like:

```suggestion
Builds are simply a large set of JSON objects that describe the supergraph. The Hasura engine then uses this description to serve your API.
```